src/smart-components/user/user-table-helpers.tsx

```diff
@@ -29,47 +31,60 @@ export interface RowProps {
     CellObject // status
   ];
   selected: boolean;
+  authModel?: boolean;
 }
 
-export const createRows = (userLinks: boolean, data: UserProps[] = [], intl: IntlShape, checkedRows = [], isSelectable = false): RowProps[] =>
-  data?.reduce<RowProps[]>((acc, { username, is_active: isActive, email, first_name: firstName, last_name: lastName, is_org_admin: isOrgAdmin }) => {
-    const newEntry: RowProps = {
-      uuid: username,
-      cells: [
-        isOrgAdmin ? (
-          <Fragment>
-            <CheckIcon key="yes-icon" className="pf-v5-u-mr-sm" />
-            <span key="yes">{intl.formatMessage(messages.yes)}</span>
-          </Fragment>
-        ) : (
-          <Fragment>
-            <CloseIcon key="no-icon" className="pf-v5-u-mr-sm" />
-            <span key="no">{intl.formatMessage(messages.no)}</span>
-          </Fragment>
-        ),
-        {
-          title: userLinks ? (
-            <AppLink to={pathnames['user-detail'].link.replace(':username', username)}>{username.toString()}</AppLink>
+export const createRows = (
+  userLinks: boolean,
+  data: UserProps[] = [],
+  intl: IntlShape,
+  checkedRows = [],
+  isSelectable = false,
+  authModel?: boolean
+): RowProps[] =>
+  data?.reduce<RowProps[]>(
+    (acc, { username, is_active: isActive, email, first_name: firstName, last_name: lastName, is_org_admin: isOrgAdmin, external_source_id }) => {
+      const newEntry: RowProps = {
+        uuid: username,
+        cells: [
+          authModel ? (
+            <OrgAdminDropdown key={`dropdown-${username}`} isOrgAdmin={isOrgAdmin} username={username} intl={intl} userId={external_source_id} />
```

Comment:
This should also be behind `isOrgAdmin` flag. Only org admins can change the org admin permissions.

Reply:
Fixed